.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,53 @@
+name: 🤖 PR auto merge
+on:
+  pull_request:
```

Comment:
Let's add additional filter to trigger it only on renovate's PRs

```suggestion
name: 🤖 PR auto merge
on:
  pull_request:
    branches:
      - renovate/**
```

What do you think ?

Reply:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-branches

Reply:
fixed